compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -62,6 +69,9 @@ SANITIZER_INTERFACE_ATTRIBUTE void __rtsan_init() {
   InitializeFlags();
   InitializeInterceptors();
 
+  if (flags().print_stats_on_exit)
+    Atexit(RtsanAtexit);
```

Comment:
Here we install our atexit handler, which may be nice for other uses later on

Reply:
I would just register `PrintStatisticsSummary` directly, if you want this to be under the `print_stats_on_exit` flag. Otherwise I would check the flag in `RtsanAtexit`